driver/flambda_backend_flags.ml

```diff
@@ -25,6 +25,8 @@ let dasm_comments = ref false (* -dasm-comments *)
 let default_heap_reduction_threshold = 500_000_000 / (Sys.word_size / 8)
 let heap_reduction_threshold = ref default_heap_reduction_threshold (* -heap-reduction-threshold *)
 
+let allow_long_frames = ref true        (* -no-long-frames *)
```

Comment:
I cannot remember exactly what we
discussed off-line: this is temporary
and the flag/default with eventually
be the other way around, right?

Reply:
Yes, this is temporary. The flag preserves the existing behavior. @mshinwell can we merge this PR as is? 
We can turn on the long frames by default in a follow up PR, after @stedolan reviews the runtime part, and more testing with a new `-dlong-frames n`.